### ISSUE TITLE
RegL1loss problem

### ISSUE THREAD

**Issue by ujsyehao** [xingyizhou/CenterNet#647]
Hi, in class RegL1Loss
`mask = mask.unsqueeze(2).expand_as(pred).float()
loss = loss / (mask.sum() + 1e-4)
`
mask.sum() is always  2 times of positive samples(center points), ex: num_pos is 2, mask.sum() return 4.
the normalization factor is different from paper.

**Comment by xingyizhou** [comment]
Thank you for pointing this out! Normalizing by `mask.sum()` is desired, since we are summing up all dimensions. We will revise the paper in the next revision.

**Comment by ujsyehao** [comment]
@xingyizhou Your opinion is we should use 2 * positive samples to normalize? Is there some thoughts about it?

**Comment by xingyizhou** [comment]
I have no strong opinions on that. It should be equivalent to set a different wh_weight.

**Comment by ujsyehao** [comment]
oh oh
